.github/workflows/update-pulp-packages.yml

```diff
@@ -39,7 +37,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-      matrix: ${{ fromJson(needs.generate_package_list.outputs.matrix) }}
+      matrix:
+        include: ${{ fromJson(needs.generate_package_list.outputs.matrix) }}
```

Comment:
This line got added with https://github.com/theforeman/pulpcore-packaging/commit/ee9da04cd5392907985023c8e093b0df9ca0d3f7 we are good to drop here